src/css/_variables.scss

```diff
@@ -4,7 +4,6 @@ $secondary-text: #F4A460; // currently just used for visited links in errors and
 $primary-bg: #000;
 $secondary-bg: lighten($primary-bg, 35%);
 
-$icon-font-family: 'VideoJS';
```

Comment:
Should we not still set this here anyway? How might someone override it with their own? If that's a real use case.